photutils/psf/tests/test_core.py

```diff
@@ -98,9 +105,12 @@ def test_psf_adapter(moffimg, prepkwargs, tols):
     f = LevMarLSQFitter()
 
     # a close-but-wrong "guessed Moffat"
-    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01, amplitude=mof.amplitude*1.01, alpha=4.79)
+    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01,
+                            amplitude=mof.amplitude*1.01, alpha=4.79)
     if prepkwargs['renormalize_psf']:
-        guess_moffat.amplitude = 5.  # definitely very wrong, so this ensures the re-normalization stuff works
+        guess_moffat.amplitude = 5.
+        # definitely very wrong, so this ensures the re-normalization
+        # stuff works
```

Comment:
This comment should go before the line `guess_moffat.amplitude = 5.`, not after it.